examples/saul/main.c

```diff
@@ -19,14 +19,56 @@
  */
 
 #include <stdio.h>
+#include <string.h>
```

Comment:
I don't think upstream RIOT administrators will allow you to add the LIS2DH12 as dependency to the general saul example. Please remove these changes.

Reply:
`tests/driver_lis2dh12` would be a better place for this.